ansible_setup/todo-ansible-playbook.yml

```diff
@@ -0,0 +1,67 @@
+- name: Install todo app on server
+  hosts: Webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+  - name: TRELLO_TOKEN
+    prompt: What is your Trello API Token?
+    private: false
```

Comment:
This is fine here, but on a real project we'd likely want this to be true even if it's not important here - I can safely say we'll be putting the Ansible down afterwards, but in reality someone would no doubt move this to another environment where it might matter (like a more accessible CI platform) without checking that the secrets have been handled secretly!

(Albeit I usually take the same shortcut, because checking you've copy-pasted it in right is much easier!)